content/working_with/console/widgets/managers.md

```diff
@@ -1,32 +1,36 @@
 ---
 layout: bt_wiki
-title: Cloudify Managers Management
+title: Spire Manager
 category: Cloudify Console
 draft: false
 ---
 
-Displays the list of the deployments created using [Cloudify Manager of Managers plugin](https://github.com/Cloudify-PS/manager-of-managers) in the current tenant, according to the user’s permissions. The data is displayed in a table.
+Displays the list of the deployments created using [Cloudify Spire plugin](https://github.com/cloudify-cosmo/cloudify-spire-plugin) in the current tenant, according to the user’s permissions. The data is displayed in a table.
 
-![managers]( /images/ui/widgets/managers.png )
+![spire-manager]( /images/ui/widgets/spire-manager.png )
```

Comment:
The Spire manager png should include more than one created manager. I realize this is challenging resource-wise. can you mock it?

Reply:
Fixed using mock.